deploy/kustomize/base/04_catalog_source_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: ${CSC_SOURCE}
+  namespace: ${MARKETPLACE_NAMESPACE}
+spec:
+  source: ${APP_REGISTRY}
```

Comment:
in https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/deploy_imageregistry.sh#L57
we have also a mechanism to deploy from a local grgp registry without the need to relay on an external appregistry.
Currently we are using it for the disconnected install workaround.
Can we implement that as well?

Reply:
Implemented using two overlays.
- overlay **marketplace** - deploys resources related to deployment via olm marketplace (e.g. catalogsourceconfig, operatorsource and secret [if needed]).
- overlay **image_registry** - deploys resources for direct image registry deployment (e.g. catalogsource). 
overlay selection is passed as an argument to `deploy_kustomize.sh`, I'll document it.